opsdroid/cli/utils.py

```diff
@@ -72,7 +86,16 @@ def configure_lang(config):
 
 
 def check_dependencies():
-    """Check for system dependencies required by opsdroid."""
+    """Check for system dependencies required by opsdroid.
+
+    Args:
+        None
```

Comment:
I think since there is no Args it's better to remove this line and just leave the Returns bit

Reply:
Sure thing